package.json

```diff
@@ -140,9 +140,9 @@
     "nock": "^11.0.7",
     "parse-link-header": "~1.0.1",
     "pkg": "^4.4.0",
-    "prettier": "1.17.0",
+    "prettier": "^1.19.1",
```

Comment:
This matches the version of `prettier` that the newer `resin-lint` version (3.1.1) depends on. Reminder: the CLI has a `npm-shrinkwrap.json` file, therefore the specification `"^1.19.1"` does not mean that `prettier` will auto update to newer versions whenever `npm install` is run.